docs/src/testing/running-tests.md

```diff
@@ -23,8 +24,9 @@ Tests: 3 passed, 0 failed, 0 skipped, 0 ignored, 0 filtered out
 ## Filtering Tests
 
 You can pass a filter string after the `snforge test` command to filter tests.
-By default, any test with an [absolute module tree path](https://book.cairo-lang.org/ch07-03-paths-for-referring-to-an-item-in-the-module-tree.html#paths-for-referring-to-an-item-in-the-module-tree)
- matching the filter will be run.
+By default, any test with an [absolute module tree path](https://book.cairo-lang.org/ch07-03-paths-for-referring-to-an-item-in-the-module-tree.html#paths-for-referring-to-an-item-in-the-module-tree) matching the filter will be run.
+
+You're tests may differ, note that we changed the default tests generated by `snforge init` for 3 custom tests: `calling`, `calling_another`, and `executing` just for the sake of this example.
```

Comment:
Is it true? I can't find it

Reply:
In fact I removed this section, I don't think it's worth mentioning at all as it may just confuse user.